valkeymodule-rs-macros-internals/src/api_versions.rs

```diff
@@ -30,6 +30,7 @@ lazy_static::lazy_static! {
         (60200, "min-redis-compatibility-version-6-2".to_string()),
         (70000, "min-redis-compatibility-version-7-0".to_string()),
         (70200, "min-redis-compatibility-version-7-2".to_string()),
+        (80000, "min-redis-compatibility-version-8-0".to_string()),
```

Comment:
@madolson - should we change it to `min-valkey-compatibility-version-8-0`?

Reply:
Ideally yes, we would migrate everything to use valkey terminology. For our usage, we could even remove all of the legacy redis api_versioning, if we intend to only support valkey.

Reply:
I see `RedisModule_AddACLCategory` in Redis 7.4 code base.  But I do like the idea of switching to `min-valkey-compatibility-version-8-0`.  

@zackcam - your thoughts?

Reply:
Yes I think switching to min-valkey-compatibility-version-8-0 seems reasonable.